webapp/src/shared/components/Row/Row.tsx

```diff
@@ -95,9 +99,9 @@ export default function Row({
         {RowChildren}
       </label>
       <input
+        type="file"
```

Comment:
why did you move this three lines above? 😯

Reply:
I think this is a leftover of me trying to add support for adding a local directory path to update the image, but it was tricky and unnecessary at the end